ql/src/java/org/apache/hadoop/hive/ql/parse/DDLSemanticAnalyzer.java

```diff
@@ -2741,7 +2741,7 @@ private void analyzeAlterTableRenamePart(ASTNode ast, String tblName,
     partSpecs.add(oldPartSpec);
     partSpecs.add(newPartSpec);
     addTablePartsOutputs(tab, partSpecs, WriteEntity.WriteType.DDL_EXCLUSIVE);
-    RenamePartitionDesc renamePartitionDesc = new RenamePartitionDesc(tblName, oldPartSpec, newPartSpec);
+    RenamePartitionDesc renamePartitionDesc = new RenamePartitionDesc(tblName, oldPartSpec, newPartSpec, null);
```

Comment:
In bootstrap load case, we just load table by table where we set the current_state_id to each objects we operate. But, the id we used is same for all objects. However, the bootstrap problem is not in the scope of this JIRA. Please create a JIRA for this issue.

Also, the DESC classes are used for both real load flow and normal DDL operations too. For normal operations, we shouldn't check the event ID. Hence passing null.

If I didn't understand your comment correctly, please revert to me.